mdp/nodes/sfa_nodes.py

```diff
@@ -369,6 +371,66 @@ def get_eta_values(self, t=1):
         return self._refcast(t / (2 * numx.pi) * numx.sqrt(self.d))
 
 
+class UnevenlySampledSFANode(SFANode):
```

Comment:
In the docstring, point out the usecase of using SFA on data with missing values. I think missing data is really the main usecase of this node. Also point out that the linear interpolation here is only the first step and to serve as bottom line for testing potential future versions that feature better interpolation ideas (there is kind of a research field around estimating covariance with missing data).

Reply:
The same thing counts for the derivative I believe. But this topic might call for a practical study on the effects of different approximation schemes in "typical" SFA problems. There could also be a generalized SFA with approximation schemes as a parameter.

Reply:
If we get it stable in this new node we can certainly think about backporting various things to the plain SFA node.